src/V3EmitC.cpp

```diff
@@ -1814,13 +1814,22 @@ class EmitCImp final : EmitCStmts {
             splitSizeInc(1);
             if (dtypep->isWide()) {  // Handle unpacked; not basicp->isWide
                 string out;
-                if (zeroit) {
-                    out += "VL_ZERO_RESET_W(";
+                if (varp->valuep()) {
+                    AstConst* const constp = VN_CAST(varp->valuep(), Const);
+                    if (!constp) varp->v3fatalSrc("non-const initializer for variable");
+                    for (int w = 0; w < varp->widthWords(); ++w) {
+                        out += varp->nameProtect() + suffix + "[" + cvtToStr(w) + "] = ";
+                        out += cvtToStr(constp->num().edataWord(w)) + "U;\n";
+                    }
                 } else {
-                    out += "VL_RAND_RESET_W(";
+                    if (zeroit) {
+                        out += "VL_ZERO_RESET_W(";
+                    } else {
+                        out += "VL_RAND_RESET_W(";
+                    }
```

Comment:
```suggestion
                    out += zeroit ? "VL_ZERO_RESET_W(" : "VL_RAND_RESET_W(";
```

Reply:
This is existing code that's not related to my change, but ok